src/client/AM.PA.MonitoringTool/SlackTracker/Data/SlackModel/UserActivity.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SlackTracker.Data.SlackModel
+{
+    class UserActivity
+    {
+        public DateTime time { get; set; }
```

Comment:
Similar to methods, in C# public Properties should start with UpperCase-letters (e.g. time -> Time). please refactor. thanks!